src/olympia/addons/tests/test_models.py

```diff
@@ -2948,8 +2949,10 @@ def test_deny_resubmission(self):
     def test_deny_already_denied_guid(self):
         addon = addon_factory()
         addon.deny_resubmission()
-        with pytest.raises(RuntimeError):
+        with pytest.raises(GuidAlreadyDeniedError) as exc_info:
             addon.deny_resubmission()
+        # Exception raised is also a child of the more generic RuntimeError.
+        assert isinstance(exc_info.value, RuntimeError)
```

Comment:
No. I wanted to make sure it's both a `GuidAlreadyDeniedError` for when we want to specifically catch that exception (like in blocklisting), and still a `RuntimeError` for more generic cases (like in add-on deletion, which doesn't care about the reason it's failing to deny since it ignores it)